applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -104,16 +104,16 @@ static struct bt_bap_lc3_preset lc3_preset_sink = BT_BAP_LC3_UNICAST_PRESET_NRF5
 static struct bt_bap_lc3_preset lc3_preset_source = BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SOURCE;
 
 static uint8_t bonded_num;
+static uint8_t num_caps;
+static uint8_t num_eps;
 static bool playing_state = true;
 
 static void ble_acl_start_scan(void);
 static bool ble_acl_gateway_all_links_connected(void);
 
-static struct bt_bap_unicast_client_discover_params audio_sink_discover_param[CONFIG_BT_MAX_CONN];
+static int temp_cap_index_get(struct bt_conn *conn, uint8_t *index);
```

Comment:
The temp_cap_index_get function can be moved up in this file, that way you don't need to declare it here.

Reply:
Didn't do this to reduce the number of differences